### PR TITLE
Mes 3666 pass debrief page add cat be

### PR DESCRIPTION
**Pull request by appleabs**
## Description
Category BE created for pass debrief page and common components extracted. 
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/pages/pass-finalisation/cat-be/_tests_/pass-finalisation.cat-be.page.spec.ts [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+import { ProvisionalLicenseReceived, ProvisionalLicenseNotReceived, PassCertificateNumberChanged } from
+  '../../../../modules/tests/pass-completion/pass-completion.actions';
+import { GearboxCategoryChanged } from '../../../../modules/tests/vehicle-details/vehicle-details.actions';
+import { D255Yes, D255No, DebriefWitnessed, DebriefUnwitnessed } from
+  '../../../../modules/tests/test-summary/test-summary.actions';
+import { CandidateChoseToProceedWithTestInWelsh, CandidateChoseToProceedWithTestInEnglish } from
+  '../../../../modules/tests/communication-preferences/communication-preferences.actions';
+import { PassFinalisationCatBEPage } from '../pass-finalisation.cat-be.page';
+
+describe('PassFinalisationPage', () => {
```
Comment (TomBillingtonUK) [inline src/pages/pass-finalisation/cat-be/_tests_/pass-finalisation.cat-be.page.spec.ts]: can you change this to include BE

File: src/pages/pass-finalisation/cat-be/pass-finalisation.cat-be.page.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+page-pass-finalisation {
```
Comment (TomBillingtonUK) [inline src/pages/pass-finalisation/cat-be/pass-finalisation.cat-be.page.scss]: You need to change this to `pass-finalisation-cat-be-page` else we will lose the styling

**Review by TomBillingtonUK — APPROVED** [review]
:)

File: src/pages/pass-finalisation/cat-be/pass-finalisation.cat-be.page.ts [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+  transmissionAutomaticRadioChecked$: Observable<boolean>;
+  transmissionManualRadioChecked$: Observable<boolean>;
+  d255$: Observable<boolean>;
+  debriefWitnessed$: Observable<boolean>;
+  conductedLanguage$: Observable<string>;
+}
+
+@IonicPage()
+@Component({
+  selector: 'page-pass-finalisation-cat-be-page',
```
Comment (TomBillingtonUK) [inline src/pages/pass-finalisation/cat-be/pass-finalisation.cat-be.page.ts]: There are 2 `pages` in the selector

**Review by MarkPatt89 — APPROVED** [review]

**Pull request merged**